plugins/techdocs/src/reader/transformers/addGitFeedbackLink.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { Transformer } from './index';
+
+// requires repo
+export const addGitFeedbackLink = (): Transformer => {
+  return dom => {
+    const contentContainer = dom.querySelector(
+      '.md-content__inner',
```

Comment:
The risk with relying too much on MkDocs material classes is that this will break as soon as we upgrade to a new version that possibly include changes to their classes (or someone are using an older/newer version that does not have this specific class). It would be nice if we can have power over this feature to 100% without relying on classes from mkdocs material.

Reply:
Could try using non-class identifiers, i'll see what can be done.

Reply:
Updated